packages/redux/src/bags/selectors.ts

```diff
@@ -711,3 +711,27 @@ export const isBagOperationLoading = (
   state: StoreState,
   bagOperationId: BagOperation['id'],
 ) => getIsBagOperationLoading(state.bag as BagsState)[bagOperationId];
+
+/**
+ * Retrieves current promocodes information in bag.
+ *
+ * @param state - Application state.
+ *
+ * @returns - Bag PromoCodes Information, `undefined` if there are no bag id
+ *
+ * @example
+ * ```
+ * import { getBagPromoCodesInformation } from '@farfetch/blackout-core/bags/redux';
```

Comment:
I have these changes in another PR but all examples should use the import from the root of the package like this:

```suggestion
 * import { getBagPromoCodesInformation } from '@farfetch/blackout-core';
```

Reply:
I think it is `@farfetch/blackout-redux`, or am I wrong?

Reply:
You are right @tiagodicosta 
Fixed 👍